electron/main.development.js

```diff
@@ -56,8 +56,27 @@ let terminateAboutWindow = false;
 const isDev = process.env.NODE_ENV === 'development';
 const isProd = process.env.NODE_ENV === 'production';
 const isTest = process.env.NODE_ENV === 'test';
+const isEtcApi = process.env.API === 'etc';
+const mantisPath = process.env.MANTIS_PATH || null;
 const daedalusVersion = process.env.DAEDALUS_VERSION || 'dev';
 
+if (isEtcApi && mantisPath) {
+  const { spawn } = require('child_process');
+  const psTree = require('ps-tree');
+
+  // Start Mantis
+  const mantis = spawn('sbt run', ['-mem 4096'], { cwd: mantisPath, detached: true, shell: true });
```

Comment:
Is it practical to change `sbt run` for something passed in from the command line...?

Reply:
I'm not sure if that is necessary since we will always be starting Mantis bat/cmd file in the future:
```
(windows) %MANTIS_PATH%/bin/mantis.bat
(macos/linux) $MANTIS_PATH/bin/mantis
```
...but I could be mistaking...

Reply:
@mcsherrylabs I have updated the code so that now the command itself is passed as `ENV` variable (along with the `MANTIS_PATH`):
```
API=etc MANTIS_PATH={path-to-mantis} MANTIS_CMD={mantis-start-command} MANTIS_ARGS={mantis-start-command-arguments} npm run dev
```

### ENV variables:
`API` - mandatory (should be set to `etc`)
`MANTIS_CMD` - mandatory
`MANTIS_PATH` - mandatory
`MANTIS_ARGS` - optional

cc @DominikGuzei @darko-mijic